README.md

```diff
@@ -24,8 +24,55 @@
 
 Several quick start options are available:
 
-- Clone the repo: `git clone https://github.com/Orange-OpenSource/ods-charts.git`
-- Install with [npm](https://www.npmjs.com/): `npm install ods-charts`
+- npm installation:
+  - Install with [npm](https://www.npmjs.com/): `npm install ods-charts`
+  - Use in your js files : `import * as ODSCharts from 'ods-charts';`
```

Comment:
```suggestion
  - Use in your JavaScript files: `import * as ODSCharts from 'ods-charts';`
```

Reply:
Maybe here we could redirect to https://apache.github.io/echarts-handbook/en/basics/import/